api/comments_api.py

```diff
@@ -114,22 +122,26 @@ def do_post(self, **kwargs) -> dict[str, str]:
           feature, gate_id, post_to_thread_type, user.email(), comment_content)
 
     # Callers don't use the JSON response for this API call.
-    return {'message': 'Done'}
+    return SuccessMessage(message='Done').to_dict()
 
   def do_patch(self, **kwargs) -> dict[str, str]:
-    comment_id = self.get_param('commentId', required=True)
-    comment: Activity = Activity.get_by_id(comment_id)
+    json_data = self.request.json
```

Comment:
@jrobbins @DanielRyanSmith Do we want to do such parsing to cast it to the auto-generated model or just keep what it was previously?

Reply:
Yes, I'd like to see us use the auto-generated model for the request parameters.